charts/backingservices/values.yaml

```diff
@@ -61,6 +61,10 @@ srs:
     # Default certificatePassword value will be empty if not used.
     # certificateName: "Certificate_Name"
     # certificatePassword: "password"
+    # To specify a certificate using a secret, uncomment the esCertsSecret parameter and provide the secret name containing your certificate and certificate password (if any).
+    # Use the full name of the certificate file (together with file extension, for example, “certificate.p12” or “certificate.jks”) as a key name in the secret. Use this key name
+    # to configure the “certificateName” parameter. Use a key name “password” to provide the certificate password in the secret.
+    # esCertsSecret: srs-certificates
```

Comment:
"es" is redundant.. probably use `CertsSecret` name

Reply:
Addressed this in latest commit